src/update_agol_vehicles_pallet.py

```diff
@@ -118,23 +109,25 @@ def update_agol_feature_service(self, sharing_map, layer,
                                 service.
         '''
 
-        sharing_draft = sharing_map.getWebLayerSharingDraft(
-            'HOSTING_SERVER', 'FEATURE', feature_service_name, [layer])
+        for item in [sddraft_path, sd_path]:
+            if arcpy.Exists(item):
+                self.log.info(f'Deleting {item} prior to use...')
+                arcpy.Delete_management(item)
```

Comment:
**suggestion**: if these are normal paths to non spatial features you don't need to involve arcpy here.

**question**: If everything is in a scratch folder, can you guarantee that it is empty at the beginning of processing rather than clean up after it. That would mean a simple delete the entire thing rather than certain items?